.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Here are some checklists you may like to use. Use your judgement.
+
+If you're new to the codebase, or you have questions, CONTRIBUTING has many answers.
+-->
+Pre-submit checklist:
+- [ ] Key commits have useful messages
+- [ ] Relevant tickets are mentioned in commit messages
+- [ ] Useful pull request description
+- [ ] Reviewer requested
+- [ ] Self-reviewed the diff
+- [ ] If you updated any cabal files or added Haskell packages:`$(nix-build default.nix -A dev.scripts.updateMaterialized)` to update the materialized Nix files
+- [ ] If you changed any Haskell files: `$(nix-shell shell.nix --run fix-stylish-haskell)` to fix any formatting issues
```

Comment:
Packages: write cradles? Or is that its own sublist.

Reply:
If you changed and Purescript files `$(nix-build -A dev.scripts.fixPurty)/bin/fix-purty`

Reply:
Yes to both, and I'll add corresponding bits to CONTRIBUTING